resources/lang/de.po

```diff
@@ -603,12 +681,9 @@ msgstr ""
 #~ msgstr "Finde im Anhang"
 
 #~ msgid "Receiver of the delivery note"
-#~ msgstr "Empfänger des Lieferscheins"
+#~ msgstr "EmpfĂ¤nger des Lieferscheins"
```

Comment:
@coditive-marcin-p It looks like we have problems with umlauts. Can you check that and fix it for the next release?

Reply:
@noahmenzi Yes, I will do that!